### DISCUSSION TITLE
"Detatch" the title from the permalink?

### DISCUSSION THREAD

**Discussion by arjolpanci-nm** [area17/twill#1257] [Q&A] [answered]
On the update/create form, whenever you change the title field, the permalink field is automatically updated to reflect the changes of the title field. In some cases, I don't really want the permalink to be based on the title of the entry, so this is a bit annoying.

During creation, it's not much of a problem since I can just edit the permalink to what I want before clicking on "Add New".

It is however a little annoying when trying to update only the title of a bunch of entries. That's because as soon as I change the title, the permalink field changes automatically, so I have to go there and set it back to what it was.

It's not a big problem, but in my case, it would save some time during bulk updates.

Is there a way to disable this behaviour? (Auto update of the permalink field when the title field changes in the create/update form)

**Comment by pboivin** [comment, marked as answer]
Hi @arjolpanci-nm. Yes, this is possible! You can unlink the 2 fields completely or choose to keep them linked only when creating a new item.

See: https://github.com/area17/twill/discussions/983

**Reply by arjolpanci-nm** [reply]
Thank you!
